openedx/core/djangoapps/timed_notification/tasks.py

```diff
@@ -0,0 +1,63 @@
+import logging
+
+from datetime import datetime, timedelta, date
+from pytz import utc
+
+from core import send_course_notification_email, get_course_link
+from lms.djangoapps.branding import get_visible_courses
+
+from celery.task import periodic_task
+from celery.schedules import crontab
+
+from django.db import close_old_connections
+
+log = logging.getLogger('edx.celery.task')
+
+
+@periodic_task(run_every=crontab(minute=1, hour=0))
+def execute_dialy():
+    log.info("TIMED-NOTIFICATION-JOB execute_dialy")
+    close_old_connections()
```

Comment:
I'm concerned with this statement. Please did down the reason for this. It appears it is something related to your environment.

Reply:
Yes, you're right, it's related to my environment. So, should I delete it ?